editoast/src/modelsv2/rolling_stock_model.rs

```diff
@@ -0,0 +1,128 @@
+use actix_web::web::Data;
+use derivative::Derivative;
+use diesel::result::Error as DieselError;
+use diesel::{ExpressionMethods, QueryDsl, SelectableHelper};
+use diesel_async::{AsyncPgConnection as PgConnection, RunQueryDsl};
+use diesel_json::Json as DieselJson;
+use editoast_derive::ModelV2;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use utoipa::ToSchema;
+use validator::{Validate, ValidationError};
+
+use crate::error::Result;
+use crate::models::rolling_stock::rolling_stock_livery::RollingStockLiveryMetadata;
+use crate::schema::rolling_stock::{
+    EffortCurves, EnergySource, Gamma, RollingResistance, RollingStockMetadata,
+    RollingStockWithLiveries,
+};
+use crate::DbPool;
+
+#[derive(Clone, Debug, Deserialize, Serialize, ModelV2, Derivative, ToSchema, Validate)]
+#[derivative(Default, PartialEq)]
```

Comment:
```suggestion
#[derivative(PartialEq)]
```

It's a modelization error to derive Default here because of the ID.